syft/frameworks/torch/differential_privacy/pate.py

```diff
@@ -350,7 +350,8 @@ def compute_q_noisy_max_torch(counts, noise_eps):
 
     _, winner = counts.max(0)
     counts_normalized = noise_eps * (
-        torch.tensor(counts, dtype=torch.float) - torch.tensor(counts[winner], dtype=torch.float)
+        counts.clone().detach().type(torch.float)
+        - torch.tensor(counts[winner].item(), dtype=torch.float)
```

Comment:
Can you also `torch.tensor(counts[winner].item(), dtype=torch.float)` -> `counts[winner].clone().detach().type(torch.float)`  or it doesn't work?
Alos you might be able to call `counts = counts.float()` before to avoir type transformation

Reply:
counts[winner] is a single integer value. that's why calling transforming it to a float (not tensor) and then creating a tensor works. will check whether creation of tensor is necessary.